Makefile

```diff
@@ -235,11 +235,16 @@ test_offline: gen_grpc 					## Run offline tests. Run with `make test_offline pr
 	-mkdir coverage
 	go test -v -parallel 1000 -timeout 60m -coverpkg=./... -coverprofile coverage/cover.out.tmp ./provider/... --tags=offline --provider=$(provider)
 
-test_offline_spark: gen_grpc 					## Run spark tests. Run with `make test_offline provider=(memory | postgres | snowflake | redshift | spark )`
+test_offline_spark: gen_grpc 					## Run spark tests.
 	@echo "These tests require a .env file. Please Check .env-template for possible variables"
 	-mkdir coverage
 	go test -v -parallel 1000 -timeout 60m -coverpkg=./... -coverprofile coverage/cover.out.tmp ./provider/... --tags=spark
 
+test_offline_k8s:  					## Run k8s tests.
```

Comment:
can we add the tests for the python script as well?
```
test_pandas:
	pytest -v -s --cov=offline_store_pandas_runner provider/scripts/k8s/tests/ --cov-report term-missing
```

Reply:
Added